adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -72,12 +72,18 @@
 MQTT_PKT_TYPE_MASK = const(0xF0)
 
 
+CONNACK_ERROR_INCORRECT_PROTOCOL = const(0x01)
+CONNACK_ERROR_ID_REJECTED = const(0x02)
+CONNACK_ERROR_SERVER_UNAVAILABLE = const(0x03)
+CONNACK_ERROR_INCORECT_USERNAME_PASSWORD = const(0x04)
+CONNACK_ERROR_UNAUTHORIZED = const(0x05)
```

Comment:
Do you want these to be available to the end-user, or are they strictly internal? If you make them start with an underscore, then they will disappear and will not take space in RAM or the .mpy file.

Reply:
I could imaging wanting to catch the error `CONNACK_ERROR_INCORECT_USERNAME_PASSWORD` to show a specific error and not need to parse the Exception...

Reply:
@dhalbert did my above comment not come through?

Reply:
Weirdly, no. Got it, use case understood.